infoschema/tables.go

```diff
@@ -1291,6 +1292,30 @@ type ServerInfo struct {
 	StartTimestamp int64
 }
 
+func (s *ServerInfo) isLoopBackOrUnspecifiedAddr(addr string) bool {
+	return strings.Contains(addr, "0.0.0.0") ||
```

Comment:
maybe better

```
ip := net.ParseIP(host)
return ip != nil && (ip.IsUnspecified() || ip.IsLoopback())
```

and it can work in ipv6

Reply:
Great, done.